inference-engine/src/inference_engine/ie_plugin_dispatcher.cpp

```diff
@@ -36,7 +36,8 @@ std::string getPluginName(const std::string& deviceName) {
     static std::map<std::string, std::string> plugunFromDeviceMap = {
         {"CPU", "MKLDNNPlugin"},    {"GPU", "clDNNPlugin"},         {"FPGA", "dliaPlugin"},
         {"MYRIAD", "myriadPlugin"}, {"HDDL", "HDDLPlugin"},         {"GNA", "GNAPlugin"},
-        {"HETERO", "HeteroPlugin"}, {"MULTI", "MultiDevicePlugin"}, {"KMB", "kmbPlugin"}};
+        {"HETERO", "HeteroPlugin"}, {"MULTI", "MultiDevicePlugin"}, {"KMB", "kmbPlugin"},
+        {"PlaidML", "PlaidMLPlugin"}};
```

Comment:
every other one is ALLCAPS may PLAIDML or PML?

Reply:
I wonder what a `plugun` is :)